.github/workflows/release-channels.yml

```
@@ -0,0 +1,52 @@
+name: set ZITADEL stable image
```

```suggestion
name: ZITADEL Release tags
```

Since it might propagate multiple channels if feel it should be agnostic

---

fine with me